conf/standalone.conf

```diff
@@ -1293,3 +1293,5 @@ disableBrokerInterceptors=true
 
 # Whether retain null-key message during topic compaction
 topicCompactionRetainNullKey=false
+
+compactionServiceFactoryClassName=org.apache.pulsar.compaction.EventTimeCompactionServiceFactory
```

Comment:
I guess this isn't intended to be part of the change?

Reply:
Added for tests, it should not be part of the change. removed.